src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementFormView.tsx

```diff
@@ -188,17 +189,17 @@ const ReimbursementRequestFormView: React.FC<ReimbursementRequestFormViewProps>
                   name="expenseTypeId"
                   control={control}
                   render={({ field: { onChange, value } }) => (
-                    <Select
-                      onChange={(newValue) => onChange(newValue.target.value)}
-                      value={value}
-                      error={!!errors.expenseTypeId}
-                    >
-                      {allExpenseTypes.map((expenseType) => (
-                        <MenuItem key={expenseType.expenseTypeId} value={expenseType.expenseTypeId}>
-                          {expenseType.name}
-                        </MenuItem>
-                      ))}
-                    </Select>
+                    <Autocomplete
```

Comment:
ok so, you should use the NERAutocomplete component, but like lowkey, I like the way this one looks better 💀 . If you're cool with this ticket getting a lil more involved, could you change the NERAutocomplete to look like this then use it in this spot instead